UPGRADE.md

```diff
@@ -65,13 +65,14 @@ We have split the `ClusterRole` for the control plane into two parts:
 
 By default, a `ClusterRoleBinding` is used to grant write permissions to the control plane, and no action is required from the user. However, if you want the control plane to have access only in specific namespaces, you can use the `namespaceAllowList` configuration to define where it should have write permissions.
 
-### Namespaces that are part of the Mesh requires `kuma.io/sidecar-injection` label to exist
+### Namespaces that are part of the mesh requires `kuma.io/sidecar-injection` label to exist
 
-Since version 2.11.x, to improve performance and security, each namespace participating in the Mesh is required to have the `kuma.io/sidecar-injection` label set.
+Since version 2.11.x, to improve performance and security, each namespace participating in the mesh is required to have the `kuma.io/sidecar-injection` label set.
 
-Before upgrading, check whether any deployments are using the `kuma.io/sidecar-injection: true` or `enabled` label in namespaces that do not have the `kuma.io/sidecar-injection` label set. If so, add `kuma.io/sidecar-injection: false` to those namespaces.
+Before upgrading, check whether any deployments are using the `kuma.io/sidecar-injection: true` or `enabled` label in namespaces that do not have the `kuma.io/sidecar-injection` label set. If so, add `kuma.io/sidecar-injection: disabled` to those namespaces.
```

Comment:
Changed `kuma.io/sidecar-injection: false` to kuma.io/sidecar-injection: disabled` to align with the script underneath.